src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSwitchExpressions.java

```diff
@@ -113,9 +113,9 @@ void howMany7(Nums k) {
     void howMany8(Nums k) {
         switch (k) {
             case ONE -> System.out.println("case two");
-            case TWO, THREE -> {} // ok, empty blocks are allowed for cases
+            case TWO, THREE -> {} // violation, 'Must have at least one statement'
```

Comment:
Text was added in https://github.com/checkstyle/checkstyle/issues/8662 . `CASE` was a token of the check then, so I am slightly confused why we originally said this was ok to be allowed.

@nrmancuso When you review.

Reply:
> I am slightly confused why we originally said this was ok to be allowed.

me too,  and in the same file ` case : { } ` is a violation

It was added based on this review https://github.com/checkstyle/checkstyle/pull/8697#discussion_r471072392

> This behavior is (poorly) [documented](https://checkstyle.sourceforge.io/config_blocks.html#EmptyBlock), let's leave it as it is.

Reply:
We don't have to have an exhaustive list in these cases, do we? I could see those not needing anything in the blocks but I don't think adding documentation would be a bad idea.

Reply:
> We don't have to have an exhaustive list in these cases

No, this is true in switch expressions only.

> I don't think adding documentation would be a bad idea.

Updating example 3 with some cases and default with arrow syntax is good

Reply:
Updated example 3 with more cases and default example with `->` and `:`

Reply:
> Text was added in #8662 . `CASE` was a token of the check then, so I am slightly confused why we originally said this was ok to be allowed.
> 
> @nrmancuso When you review.

Yeah this was weird, I think this should be a violation